boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
```

Comment:
We use the `quickstarter` spelling throughout ...

Reply:
done!